socorro/processor/rules/breakpad.py

```diff
@@ -124,6 +124,34 @@ def action(self, raw_crash, dumps, processed_crash, tmpdir, status):
         )
 
 
+class HasGuardPageAccessRule(Rule):
+    """Compute has_guard_page_access value
+
+    Fills in:
+
+    * has_guard_page_access (bool): whether there are "is_likely_guard_page=True"
+      in json_dump.crash_info.memory_accesses structures
+
+    """
+
+    def action(self, raw_crash, dumps, processed_crash, tmpdir, status):
+        accesses = glom.glom(
+            processed_crash, "json_dump.crash_info.memory_accesses", default=None
+        )
+        if accesses is None:
+            return
+
+        # is_likely_guard_page is True iff at least one of the memory_accesses values is
+        # true
+        has_guard_page_access = any(
+            [data.get("is_likely_guard_page", False) for data in accesses]
```

Comment:
any reason this is a list comprehension rather than just a generator (ie without the [])?

Reply:
Yes! Because I've been using Python for a half a million years and I keep forgetting you can do that. I'll fix it.